apps/dashboard/src/components/editor/index.js

```diff
@@ -21,8 +21,9 @@ import EditorLoadingPlaceholder from './loading-placeholder';
 import { editorSettings } from './settings';
 import Toolbar from './toolbar';
 import UnpublishedChangesNotice from './unpublished-changes-notice';
-import { useAutosave } from './use-autosave';
+import { useEditorContent } from './use-editor-content';
 import { useStylesheet } from '@crowdsignal/hooks';
+import { EDITOR_VIEW_AUTO, EDITOR_VIEW_DRAFT } from '../../util/editor';
```

Comment:
`../../util` is literally about to be removed in #132. Regardless of that, these should just be in `src/components/editor/constants.js` or something - don't see how anything else than the editor component would need to use these.

Reply:
Gotcha, didn't know where to put it, will move those.